nodes/core/io/31-tcpin.js

```diff
@@ -34,6 +34,7 @@ module.exports = function(RED) {
         this.server = (typeof n.server == 'boolean')?n.server:(n.server == "server");
         this.closing = false;
         this.connected = false;
+        this.cluster = native;
```

Comment:
Good catch!  I need to go through the other nodes and make sure I switched over per our new convention.